packages/hash/frontend/src/blocks/page/BlockView.tsx

```diff
@@ -35,102 +41,98 @@ type BlockHandleProps = {
-      throw new Error(`Non-block entity ${entityId} loaded into BlockView.`);
+  const updateChildEntity = (properties: JSONObject) => {
+    const childEntity = blockEntity?.properties.entity;
+    if (!childEntity) {
+      throw new Error(`No child entity on block to update`);
     }
+    blockView.manager.updateEntityProperties(childEntity.entityId, properties);
+  };
 
-    const blockView = useBlockView();
-
-    const updateChildEntity = (properties: JSONObject) => {
-      const childEntity = blockEntity?.properties.entity;
-      if (!childEntity) {
-        throw new Error(`No child entity on block to update`);
-      }
-      blockView.manager.updateEntityProperties(
-        childEntity.entityId,
-        properties,
-      );
-    };
-
-    const blockSchema = blockEntity
-      ? blocksMetaMap[blockEntity.properties.componentId]?.componentSchema
-      : null;
-
-    return (
-      <Box
-        ref={ref}
-        sx={{
-          position: "relative",
-          cursor: "pointer",
-          height: 24,
-        }}
-        data-testid="block-changer"
-      >
-        <DragVerticalIcon {...bindTrigger(contextMenuPopupState)} />
-
-        <BlockContextMenu
-          blockEntity={blockEntity}
-          blockSuggesterProps={blockSuggesterProps}
-          deleteBlock={deleteBlock}
-          entityId={entityId}
-          openConfigMenu={configMenuPopupState.open}
-          popupState={contextMenuPopupState}
-          ref={blockMenuRef}
-        />
-
-        <BlockConfigMenu
-          anchorRef={ref}
-          blockEntity={blockEntity}
-          blockSchema={blockSchema}
-          closeMenu={configMenuPopupState.close}
-          updateConfig={(properties: JSONObject) =>
-            updateChildEntity(properties)
-          }
-          popupState={configMenuPopupState}
-        />
-      </Box>
-    );
-  },
-);
+  const blockSchema = blockEntity
+    ? blocksMetaMap[blockEntity.properties.componentId]?.componentSchema
+    : null;
+
+  return (
+    <Box
+      ref={ref}
+      sx={{
+        position: "relative",
+        cursor: "pointer",
+        height: 24,
+      }}
+      data-testid="block-changer"
+    >
+      <DragVerticalIcon {...bindTrigger(contextMenuPopupState)} />
+
+      <BlockContextMenu
+        blockEntity={blockEntity}
+        blockSuggesterProps={blockSuggesterProps}
+        deleteBlock={deleteBlock}
+        entityId={entityId}
+        openConfigMenu={configMenuPopupState.open}
+        popupState={contextMenuPopupState}
+        ref={blockMenuRef}
+      />
+
+      <BlockConfigMenu
+        anchorRef={ref}
+        blockEntity={blockEntity}
+        blockSchema={blockSchema}
+        closeMenu={configMenuPopupState.close}
+        updateConfig={(properties: JSONObject) => updateChildEntity(properties)}
+        popupState={configMenuPopupState}
+      />
+    </Box>
+  );
+};
 
-BlockHandle.displayName = "BlockHandle";
+export const BlockHandle = forwardRef(BlockHandleComponent);
```

Comment:
This is inconsistent with the other `forwardRef` you've done – and it also means consumers will use this as `BlockHandle` but it'll show up in dev tools as `BlockHandleComponent` – you probably ought to follow the other approach

Reply:
Tried that approach initially. Renaming the export wouldn't work since the component is used in the same file (i.e you'd have to use `BlockHandleForwardRef`)... For consistency, I've moved BlockHandle to a separate file, so I can use that approach. Thanks! 

Updated in => [b7354c7](https://github.com/hashintel/hash/pull/506/commits/b7354c797ca5e5273c4c97000d10c68b9ac4d42e)